docs/develop/go/continue-as-new.mdx

```diff
@@ -24,9 +24,10 @@ func SimpleWorkflow(ctx workflow.Context, value string) error {
 
 To check whether a Workflow Execution was spawned as a result of Continue-As-New, you can check if `workflow.GetInfo(ctx).ContinuedExecutionRunID` is not empty (i.e. `""`).
 
-**Notes**
+:::note
```

Comment:
I'm leaving this as a non-blocking comment as it was mostly a Patrick-driven thing as far as I understand but somewhere in our various style rules there's a rule that says we always put a title in.

New regime, we can figure it out. I don't have  dog in this fight, just reporting in. Maybe check with Brian?